client/scss/components/_header_block.scss

```diff
@@ -0,0 +1,107 @@
+//* @define header-block;
+
+.header-block {
+  &.header-block--a {
+    @include respond-to('large') {
+      position: relative;
+      padding: 1.8rem 0 1.5rem span-columns('large', 1.5);
+
+      &:before {
+        content: '';
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        width: span-columns('large', 0.5);
+        background: color('keyline-grey');
+      }
+    }
+  }
+
+  &.header-block--b,
+  &.header-block--c:not(.header-block--xlarge) {
+    padding-bottom: 1.2rem;
+    border-bottom: 1px solid color('keyline-grey');
+  }
+}
+
+.header-block__title {
+  margin-top: 0;
+  margin-left: -0.06em;
+
+  .header-block--a & {
```

Comment:
This is starting to break BEM again.
Could we not set the state on the title?

i.e.
```
.header-block__title--a {}
```

Also - what is `--a`?